docs/concepts/cluster_queue.md

```diff
@@ -173,18 +238,18 @@ ClusterQueue.
 
 ### Flavors and borrowing semantics
 
-When borrowing, Kueue satisfies the following semantics:
+When borrowing, Kueue satisfies the following admission semantics:
 
-- When assigning flavors, Kueue goes through the list of flavors in
-  `.spec.resources[*].flavors`. For each flavor, Kueue attempts to
-  fit the workload using the min quota of the ClusterQueue or the unused
-  min quota of other ClusterQueues in the cohort, up to the max quota of the
-  ClusterQueue. If the workload doesn't fit, Kueue proceeds evaluating the next
+- When assigning flavors, Kueue goes through the list of flavors in the
+  ClusterQueue's `.spec.resources[*].flavors`. For each flavor, Kueue attempts
+  to fit a Workload's pod set using the `min` quota of the ClusterQueue or the
+  unused `min` quota of other ClusterQueues in the cohort, up to the `max` quota
+  of the ClusterQueue. If the workload doesn't fit, Kueue proceeds evaluating the next
   flavor in the list.
-- Borrowing happens per-flavor. A ClusterQueue can only borrow quota of flavors
-  it defines.
+- A ClusterQueue can only borrow quota of flavors it defines and it can only
+  borrow quota for one flavor.
```

Comment:
I wouldn't mention "and it can only borrow quota for one flavor" because that also applies in the no borrowing case as well.